<?xml version='1.1' encoding='UTF-8'?>
<Tag plugin="workflow-support@820.vd1a_6cc65ef33">
  <node class="cps.n.StepEndNode" plugin="workflow-cps@2725.v7b_c717eb_12ce">
    <parentIds>
      <string>43</string>
    </parentIds>
    <id>44</id>
    <startId>42</startId>
  </node>
  <actions>
    <wf.a.BodyInvocationAction plugin="workflow-api@1164.v760c223ddb_32"/>
    <wf.a.TimingAction plugin="workflow-api@1164.v760c223ddb_32">
      <startTime>1657718404630</startTime>
    </wf.a.TimingAction>
    <wf.a.ErrorAction plugin="workflow-api@1164.v760c223ddb_32">
      <error class="java.lang.InterruptedException">
        <detailMessage>[DeployPublisher][WARN] No wars found. Deploy aborted. %n</detailMessage>
        <stackTrace>
          <trace>hudson.plugins.deploy.DeployPublisher.perform(DeployPublisher.java:107)</trace>
          <trace>jenkins.tasks.SimpleBuildStep.perform(SimpleBuildStep.java:123)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.CoreStep$Execution.run(CoreStep.java:101)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.CoreStep$Execution.run(CoreStep.java:71)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.SynchronousNonBlockingStepExecution.lambda$start$0(SynchronousNonBlockingStepExecution.java:47)</trace>
          <trace>java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)</trace>
          <trace>java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)</trace>
          <trace>java.base/java.lang.Thread.run(Thread.java:829)</trace>
        </stackTrace>
        <suppressedExceptions class="empty-list"/>
      </error>
    </wf.a.ErrorAction>
  </actions>
</Tag>